doc/source/train/user-guides/local_mode.rst

```diff
@@ -0,0 +1,344 @@
+.. _train-local-mode:
+
+Local Mode
+==========
+
+.. important::
+    This user guide shows how to use local mode for Ray Train V2 only.
```

Comment:
Link to https://github.com/ray-project/ray/issues/49454, mention that v2 is enabled by default starting from ray version X, and remove the bolded line